configs/tsconfig.backend.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "types": ["corvid-types/types/backend"]
+    "typeRoots": ["../", "node_modules/@types"],
+    "types": ["types/backend", "node"]
```

Comment:
please move the changes to the 3 config files to a different commit explaining what the change does